src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,83 @@
-export const Pagination = () => {};
+import React from 'react';
+
+function calcPagesCount(totalItems: number, sortCount: string): number {
+  return Math.ceil(totalItems / Number(sortCount));
+}
+
+function createArrByNumber(num: number): number[] {
+  const arr = [];
+
+  for (let i = 1; i <= num; i++) {
+    arr.push(i);
+  }
+
+  return arr;
+}
+
+interface PagionationProps {
```

Comment:
There is a typo in the interface name `PagionationProps`. It should be `PaginationProps` to match the component name and maintain consistency.